### ISSUE TITLE
issues with baseurl and documentation

### ISSUE THREAD

**Issue by axk4545** [clarkhacks/Minimal-Text#2]
I am attempting to use your theme and am having trouble modifying the _config.yml specifically the baseurl. I attempted to find documentation on what to change but was unable too. Is there any chance you could point me in the right direction?

**Comment by clarkhacks** [comment]
Yeah no problem! The `baseurl` is used only for assets and the home button. 
```html
./_includes/header.html
<a href="{{ site.baseurl }}">[ Home ]</a>
...
./_layouts/default.html
<link rel="stylesheet" href="/Minimal-Text/css/style.css">
```
(Here an error was made and "/Minimal-Text" was not replaced with `{{ site.baseurl }}`)
If you are not using a subdomain on Gh-Pages you may change `baseurl` in `_config.yml` to "/".
Also note "/Minimal-Text" in the default layout will need to be changed as well.
```yaml
title: Minimal Text
baseurl: "/"
desc: A minimal blog for off-topic blog entries.
```
